tidy3d/plugins/design/design.py

```diff
@@ -41,31 +49,51 @@ class DesignSpace(Tidy3dBaseModel):
         ...,
         title="Search Type",
         description="Specifications for the procedure used to explore the parameter space.",
+        discriminator=TYPE_TAG_STR,  # Stops pydantic trying to validate every method whilst checking MethodType
+    )
+
+    task_name: str = pd.Field(
+        "",
+        title="Task Name",
+        description="Task name assigned to tasks along with a simulation counter in the form of {task_name}_{counter}. \
```

Comment:
dont need the `\`, we just end the string with `"` (no comma) and continue on next line and it seems to work